client/src/app.js

```diff
@@ -30,7 +30,8 @@ export default function main({ DOM, HTTP, route, storage, scanner: scan$, search
 
   /// User actions
   , page$     = route()
-  , goHome$   = route('/').map(loc => ({ start_height: loc.query.start != null ? +loc.query.start : null }))
+  , dashBoard$ = route('/')
+  , goBlocks$   = route('/recent/blocks').map(loc => ({ start_height: loc.query.start != null ? +loc.query.start : null }))
```

Comment:
We use `/tx/recent`, so I would go with `/blocks/recent` here to keep things more consistent.

Reply:
Changes have been made